### PR TITLE
jithendar93's first goodie

### PR DESCRIPTION
**Pull request by jithendar93**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
jithendar93's first goodie; works as it says in the tutorial

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It solves nothing; just a tutorial;

**What is the data source for your Instant Answer? (Provide a link if possible)**
--

**Why did you choose this data source?**
--

**Are there any other alternative (better) data sources?**
--

**What are some example queries that trigger this Instant Answer?**
--

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
--

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
--

**Which existing Instant Answers will this one supercede/overlap with?**
--

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
From a professor in my university

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
[] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

**Comment by moollaza** [comment]
@jithendar93 congrats on your first Goodie! We'll take a look and give you some feedback shortly if any changes are necessary. 

Thanks a lot for taking the time to contribute, we really appreciate it!

File: lib/DDG/Goodie/IsAwesome/jithendar93.pm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+zci is_cached   => 1;
+
+name "IsAwesome jithendar93";
+description "My first Goodie, it let's the world know that jithendar93 is awesome";
+primary_example_queries "duckduckhack jithendar93";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/jithendar93.pm";
+attribution github => ["https://github.com/jithendar93", "jithendar93"],
+#            twitter => "TwitterUserName";
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/jithendar93.pm]: Please remove this line, or fill in your twitter handle if you'd like attribution

**Comment by moollaza** [comment]
ping! @jithendar93 are you interested in finishing this up so we can merge it in? We'd love to see this go live!

**Comment by mintsoft** [comment]
Hey @jithendar93 Just checking in!

**Comment by moollaza** [comment]
Closing this for now. @jithendar93 if you'd like to finish this up, you're more than welcome to! We'd love to see it go live :+1:

**Pull request closed without merging**